apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -160,6 +196,7 @@ function LandingPage({
 
           if (response.ok) {
             const jsonData = await response.json();
+            setLoadingWorkshopsAsFacilitator(false);
```

Comment:
Do we also want to turn the spinner off in the case of a caught error as well for these?

Reply:
oh good call